src/__tests__/standalone-analytics.test.ts

```diff
@@ -220,26 +219,23 @@ describe('standalone bundle', () => {
     }, 0)
   })
   it('sets buffered event emitters before loading destinations', async (done) => {
-    // @ts-ignore ignore Response required fields
-    mocked(unfetch).mockImplementation((): Promise<Response> => fetchSettings)
+    mocked(unfetch).mockImplementation(() => fetchSettings as Promise<Response>)
 
     const operations: string[] = []
 
     track.mockImplementationOnce(() => operations.push('track'))
-    on.mockImplementationOnce(() => operations.push('on', 'on'))
+    on.mockImplementationOnce(() => operations.push('on'))
```

Comment:
@arielsilvestri -- quick santity check here?  I had to update this test you wrote after fixing the double "window.analytics.on('foo') call" bug.

Reply:
👍